### ISSUE TITLE
Sub-11: Journal Database Seeded with 10,000+ Journals

### ISSUE THREAD

**Issue by vatsal142** [vatsal142/journalai#50]
## What This Task Is

Import the full journal dataset into the production PostgreSQL database
so that JournalAI has real data to search against on launch day.

From PRD Section 11, Checklist Item 14:
  Journal database seeded with minimum 10,000 journals.

From PRD Section 7 (Search Orchestration):
  The search runs 6 parallel API calls: OpenAlex, CrossRef, DOAJ, Semantic Scholar, Scopus, Bealls List.
  Results are merged by ISSN, AI-scored, filtered, sorted, paginated.

## Schedule

| Field      | Value      |
|------------|------------|
| Start Date | 2026-09-07 |
| End Date   | 2026-09-09 |
| Owner      | Solo Dev   |

## Why This Matters

JournalAI fetches journal metadata from 5 external APIs at search time,
but journal base data (ISSN, title, publisher, subject area, Impact Factor)
needs to be in the local database for filtering and fast lookup.
Without seed data, searches return no results and filters cannot work.

### Source 1: SCImago Journal Rankings (Primary)

`
URL: https://www.scimagojr.com/journalrank.php

Steps:
1. Go to the URL above
2. Scroll to the bottom of the table
3. Click the Download CSV icon (small icon below the table)
4. Download ALL years available (or at minimum the latest year)
5. File name: scimagojr YYYY.csv
6. Expected rows: 20,000+ journals

CSV columns include:
  Rank, Sourceid, Title, Type, Issn, SJR, SJR Best Quartile,
  H index, Total Docs (2023), Total Refs, Total Cites (3years),
  Citable Docs (3years), Cites / Doc (2years), Ref / Doc, Country, Publisher
`

### Source 2: DOAJ Journal List (Open Access journals)

`
URL: https://doaj.org/csv

Steps:
1. Visit the URL above (auto-downloads a CSV)
2. This contains all DOAJ-indexed open access journals
3. Import to enrich is_doaj flag and APC data
4. File contains: journal_title, issn, eissn, apc_amount, apc_currency
`

### How to Import via Admin Panel

`
1. Open https://journalai.com/admin
2. Log in with admin credentials (set up in Sub-12)
3. Go to: Admin --> Journal Database (M-12)
4. Click: Import CSV
5. Select the SCImago CSV file
6. Preview shows first 5 rows -- confirm columns are mapped correctly
7. Click Import
8. Wait 3-5 minutes for 20,000+ rows to import
9. Success message shows: Imported X journals, Skipped Y duplicates

Repeat for DOAJ CSV:
1. Admin --> Journal Database --> Import CSV
2. Select DOAJ CSV
3. This run will update existing records with APC and OA data
`

### Verify Data Quality

`sql
-- Connect to production RDS and run these checks:

-- Total journal count (must be 10,000+):
SELECT COUNT(*) FROM journals;

-- Distribution by subject area (spot check):
SELECT subject_areas, COUNT(*) as count
FROM journal_metrics
GROUP BY subject_areas
ORDER BY count DESC
LIMIT 10;

-- Check journals have metrics:
SELECT COUNT(*) FROM journals j
JOIN journal_metrics jm ON j.id = jm.journal_id
WHERE jm.impact_factor IS NOT NULL;

-- Check predatory journals imported:
SELECT COUNT(*) FROM predatory_journals WHERE review_status = 'CONFIRMED';
`

### Import Predatory Journal List (Bealls List)

`
1. Admin --> Predatory List (M-13)
2. Click: Sync Bealls List
3. This calls the ETL pipeline from /api/admin/predatory/sync
4. Downloads latest Bealls List from GitHub
5. Imports into predatory_journals table
6. Verify: Admin should show X predatory journals flagged
`

### Run a Test Search to Confirm Data

`
1. Go to https://journalai.com/journal-recommendation
2. Paste this abstract (150+ words):
   "This study investigates the application of machine learning algorithms
    for the early detection of diabetic retinopathy in fundus photography.
    We trained a convolutional neural network on a dataset of 10,000 retinal
    images and achieved 94% sensitivity and 92% specificity. The model was
    validated on an independent test set and compared to expert grading.
    Our approach could enable scalable screening programs in resource-limited settings."
3. Click Find Journals
4. Should return at least 5 ranked journals within 5 seconds
5. Verify: AI Match % scores are shown
6. Verify: Some journals show Scopus, WoS, PubMed indexing chips
7. Verify: At least 1 journal shows predatory warning (red badge) if applicable
`

## Checklist

- [ ] SCImago CSV downloaded (latest year, 20,000+ rows)
- [ ] SCImago CSV imported via Admin --> Journal Database
- [ ] Import result shows 10,000+ journals successfully imported
- [ ] DOAJ CSV downloaded from doaj.org/csv
- [ ] DOAJ CSV imported to enrich OA and APC data
- [ ] SELECT COUNT(*) FROM journals returns 10,000+
- [ ] Journals have Impact Factor, H-Index, CiteScore populated
- [ ] Bealls List synced via Admin --> Predatory List --> Sync
- [ ] Predatory journals count shows in admin panel
- [ ] Test search returns minimum 5 results with AI Match scores
- [ ] Predatory badges showing on flagged journals
- [ ] Indexing chips (Scopus, WoS, PubMed) showing on relevant journals

## Dependencies

- Sub-03: PostgreSQL RDS and Prisma migrations must be done first
- Sub-02: Admin panel must be deployed to import data
